src/Hintman/Config.hs

```diff
@@ -44,9 +44,11 @@ hintmanConfiguationCodec = HintmanConfig
 
 loadFileConfig :: FilePath -> IO HintmanConfig
 loadFileConfig path =
-    Toml.decodeFile hintmanConfiguationCodec path `catch` catchLoad
+    (Toml.decodeFile hintmanConfiguationCodec path `catch` catchExcAndReturnDefaultConfig)
+      `catch` catchExcAndReturnDefaultConfig
   where
-    catchLoad :: LoadTomlException -> IO HintmanConfig
-    catchLoad _ = do
-        print $ "Hintman failed to load config from " <> path
-        pure defaultHintmanConfig
+    catchExcAndReturnDefaultConfig :: SomeException -> IO HintmanConfig
+    catchExcAndReturnDefaultConfig e = case e of
```

Comment:
If you write like this

```suggestion
    catchExcAndReturnDefaultConfig e = defaultHintmanConfig <$ case e of
```

you won't need to make it in every case and the `<$` operator will still work okay.

Reply:
Ooh :+1: